HW8_Descriptors and Metaclass/HW8_Descriptors and Metaclass.py

```diff
@@ -254,7 +256,8 @@ def place_order(self):
 order1 = ShoppingBag()  # создаем заказ
 order1.add_goods_to_bag('T-shirt', 10)  # добавляем товар в корзину
 order1.add_goods_to_bag('Jeans', 5)  # добавляем товар в корзину
-# print(order1._bag)
+
```

Comment:
Product and Mywarehouse should be different classes. The same is for Shopping cart and Order